### ISSUE TITLE
Attribution in image api

### ISSUE THREAD

**Issue by jpstroop** [IIIF/iiif.io#227]
Cover this when we do auth

**Comment by azaroth42** [comment]
Proposal:
  * Include attribution, logo and license into info.json
  * Copy the HTML recommendation and similar gumf into the description of attribution to keep the two documents independent

**Comment by jpstroop** [comment]
:+1: I think; I don't love the repetition, but it seems like the practical thing to do.

**Comment by azaroth42** [comment]
Example info.json ... note the potential recursion...

```javascript
{
  "@id": "http://example.edu/iiif/image/main-image",
  "height", 4000,
  "width": " 6000,
  "attribution": "<span>Provided by <a href=\"http://example.edu/\">some institution</a></span>",
  "license": "http://creativecommons.org/license/cc-by-4.0",
  "logo": {
    "@id": "http://example.edu/images/logo.jpg",
    "service": {
      // I heard you like info.json, so I put some info.json in your info.json
      "@id": "http://example.edu/iiif/image/logo",
      "profile": "http://iiif.io/api/image/2/level2.json",
      "height":  150,
      "width":  150
    }
  }
}
```

**Comment by jpstroop** [comment]
Buh. Does logo have to be a service?

**Comment by azaroth42** [comment]
No, but it may have a service. And then that service may have a logo.

**Comment by jpstroop** [comment]
OK, but just a plain ol' URI is OK too, right?

**Comment by azaroth42** [comment]
Yup.  The Presentation API text is:

A small image that represents an individual or organization associated with the resource it is attached to. This could be the logo of the owning or hosting institution. It is recommended that a IIIF Image API service be available for this image for manipulations such as resizing.

**Comment by jpstroop** [comment]
Gotcha, right.

**Comment by jpstroop** [comment]
Can my logo have a logo? :stuck_out_tongue:

**Comment by azaroth42** [comment]
Yes.

**Comment by azaroth42** [comment]
Proposed text in:
http://image-auth.iiif.io/api/image/2.1/#image-information

**Comment by jpstroop** [comment]
Bring out in spec that semantics are identical to those in the prezi API. Otherwise :+1: from tech group at GW meeting.

**Comment by jpstroop** [comment]
Closed in c3a1d2efff88d3e14e32d5f2945e5d59edc6461f
